app/views/admin/index.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <div class="column">
   <section class="top_customers">
-    <h3>  Top 5 Customers</h3>
+    <h3>  Top 5 Customers </h3>
       <ol>
         <% @top_5_customers.each do |customer| %>
           <li><%="#{customer.first_name} #{customer.last_name} has made #{customer.result_count} successful transactions"%></li><br>
```

Comment:
Is there a chance we can change the variable name `result_count` to something more intuitive? Maybe `successful_transactions`?